plugins/faustwp/includes/replacement/functions.php

```diff
@@ -131,3 +131,106 @@ function is_wp_link_ajax_request(): bool {
 		&& ! empty( $_POST['action'] )
 		&& 'wp-link-ajax' === $_POST['action'] );
 }
+
+/**
+ * Get all site URLs for each possible HTTP protocol
+ */
+function faustwp_get_wp_site_urls() {
```

Comment:
Could you please add the return type here and to the phpdoc?

Reply:
Good spot 💯

Reply:
Updated 👍